core/templates/dev/head/components/CkEditorRteDirective.js

```diff
@@ -192,7 +192,32 @@ oppia.directive('ckEditorRte', [
         }, null, null, 20);
 
         ck.on('change', function() {
-          ngModel.$setViewValue(ck.getData());
+          var elt = document.createElement('div');
+          elt.innerHTML = ck.getData();
```

Comment:
Typically we avoid using innerHTML directly, for security reasons. How about using some jQuery construct instead, like we used to do?

Reply:
Yes, done. PTAL!